app/templates/application.hbs

```diff
@@ -1,5 +1,13 @@
-{{page-title (t "utility.appName")}}
-
+{{#let (config 'environmentName') as |environmentName|}}
+  {{page-title (concat (t "utility.appName") (if environmentName (concat ' - ' environmentName)))}}
+  {{#if environmentName}}
```

Comment:
Nitpick: Would it make sense to check if the environmentName is not equal to `{{ENVIRONMENT_NAME}}` instead of a thruthy check? That way it won't be displayed by accident if the value isn't set to an empty string on the server.

Reply:
Good idea, I modified the code accordingly.